programs/token/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#[cfg(feature = "associated-token")]
+pub mod associated_token;
+// Last updated at mainnet-beta slot height: 298486032
```

Comment:
nit: can we move this to the top as a crate/module doc? ie:

```suggestion
//! Last updated at mainnet-beta slot height: 298486032

#[cfg(feature = "associated-token")]
pub mod associated_token;
```